autotest/test_gwe_drycell_conduction0.py

```diff
@@ -0,0 +1,424 @@
+# ## Test problem for GWE
```

Comment:
@wpbonelli has changed all the comments at the top of the autotests to triple quotes instead of a comment for each line, which I think is better.  Can do this after merge.

Reply:
motivation here is for pytest to recognize/emit the docstring as a description of the test case, e.g. 

```
% pytest -v --collect-only test_gwf_maw06.py
...
collected 2 items                                                                                         

<Module test_gwf_maw06.py>
  Test maw package ability to equalize the flow correction.
  maw_06a - well start at .25, aquifer starts at 2
  maw_06b - well starts at 2, aquifer starts at .25
  <Function test_mf6model[0-maw_06a]>
  <Function test_mf6model[1-maw_06b]>
```

Reply:
I didn't catch this modification earlier and should have.  Fixed with d517424